config/rviz/zed_example.rviz

```diff
@@ -466,7 +493,7 @@ Visualization Manager:
   Enabled: true
   Global Options:
     Background Color: 212; 212; 212
-    Fixed Frame: map
+    Fixed Frame: odom
```

Comment:
Double check whether this should be map or odom (idk if this should have changed or not)

Reply:
I didnt modify that manually,this is just an rviz change i think. I saved my rviz config because the topic names are different now, this change came with smoehow.

Reply:
I know that. but my point still stands. The fixed frame settings affect what frame Nav2 goals are relative to when dropping Nav2 goal poses in rviz, so we don't want to accidentally change this. I think it should be changed back to map (not odom) but it's possible that I'm wrong